java/src/Ice/src/main/java/com/zeroc/Ice/_ObjectPrxI.java

```diff
@@ -6,21 +6,23 @@
 
 import com.zeroc.IceInternal.OutgoingAsync;
 import com.zeroc.IceInternal.Reference;
+import com.zeroc.IceInternal.RequestHandlerCache;
 import java.util.LinkedList;
 import java.util.List;
 import java.util.Map;
 import java.util.concurrent.CompletableFuture;
 
 /** Concrete proxy implementation. */
 public class _ObjectPrxI implements ObjectPrx, java.io.Serializable {
-  protected _ObjectPrxI(Reference ref) {
+  public _ObjectPrxI(Reference ref) {
```

Comment:
Unfortunately, these constructors need to be made public now. They live in the `Ice` package, but are called by code in `IceInternal` now; a completely different package.

Reply:
Note a big deal since _ObjectPrxI is not really a "public" API.